.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Jekyll",
+  "image": "mcr.microsoft.com/devcontainers/jekyll:2-bullseye",
+  "onCreateCommand": "git config --global --add safe.directory ${containerWorkspaceFolder}",
+  "postCreateCommand": "bash .devcontainer/post-create.sh",
+  "postAttachCommand": "exec zsh",
```

Comment:
@kungfux Although I have OMZ enabled here, it is recommended to use vscode's _Source Control_ and _Git Graph_ instead of typing Git commands in the terminal.

Reply:
_Git Graph_ is a great extensiona sn this is a nice toolset for everyone. `tasks.json` would be a good fit to serve the overall idea later :)